app/services/exercise_service.rb

```diff
@@ -0,0 +1,20 @@
+class ExerciseService
+
+  def self.get_all_exercises
+    response = conn.get("exercises")
+    parse_json(response)
+  end
+
+  private
+
+  def self.conn
+    Faraday.new(url: "https://exercisedb.p.rapidapi.com/") do |require|
+      require.headers['X-RapidAPI-Host'] = 'exercisedb.p.rapidapi.com'
+      require.headers['X-RapidAPI-Key'] = ENV['exercise_key']
```

Comment:
Great job here - @SullyBirashk , You and I need to remember to get these keys on our local

Reply:
Thanks for Pointing that out @gregoryjflaherty